cmake/OpenCVCompilerOptimizations.cmake

```diff
@@ -343,12 +345,17 @@ elseif(ARM OR AARCH64)
     endif()
     ocv_update(CPU_FP16_IMPLIES "NEON")
   else()
-    ocv_update(CPU_KNOWN_OPTIMIZATIONS "NEON;FP16;NEON_DOTPROD")
+    ocv_update(CPU_KNOWN_OPTIMIZATIONS "NEON;FP16;NEON_DOTPROD;FP16_SIMD;BF16_SIMD")
```

Comment:
> FP16_SIMD;BF16_SIMD

How does scope of these instructions correlate with other platforms?
... and available universal intrinsics?

Until it is unclear, it is better to add `ARM_` prefix.

Reply:
fixed